Documentation/ceph-common-issues.md

```diff
@@ -813,7 +814,75 @@ If you still decide to configure an OSD on LVM, please keep the following in min
 ### Solution
 
 - Disable `lvmetad.`
-- Avoid configuration of LVs from the host. In addition, don't touch the VGs and physical volumes that back these LVs. 
+- Avoid configuration of LVs from the host. In addition, don't touch the VGs and physical volumes that back these LVs.
 - Avoid incrementing the `count` field of `storageClassDeviceSets` and create a new LV that backs a OSD simultaneously.
 
 You can know whether the above-mentioned tag exists tag with the command: `sudo lvs -o lv_name,lv_tags`. If the `lv_tag` field is empty in an LV corresponding to the OSD lv_tags, this OSD encountered the problem. In this case, please [retire this OSD](ceph-osd-mgmt.md#remove-an-osd) or replace with other new OSD before restarting.
+
+## Prepare job fails due to low aio-max-nr setting
+
+If the Kernel is configured with a low [aio-max-nr setting](https://www.kernel.org/doc/Documentation/sysctl/fs.txt), the OSD prepare job might fail with the following error:
+
+```text
+exec: stderr: 2020-09-17T00:30:12.145+0000 7f0c17632f40 -1 bdev(0x56212de88700 /var/lib/ceph/osd/ceph-0//block) _aio_start io_setup(2) failed with EAGAIN; try increasing /proc/sys/fs/aio-max-nr
+```
+
+To overcome this, you need to increase the value of `fs.aio-max-nr` of your sysctl configuration (typically `/etc/sysctl.conf`).
+You can do this with your favorite configuration management system.
+
+Alternatively, you can have a DaemonSet to apply the configuration for you on all your nodes:
+
+```yaml
+kind: DaemonSet
```

Comment:
Something doesn't feel right about recommending a daemonset to apply a system setting that is outside of rook. Instead of including this daemonset in the docs, what if we just link to the example in the issue comment as one way to do it?

Reply:
As you said it's outside of Rook, so I don't see this daemonset approach being an issue but I'll link the issue instead.